4_inspect.R

```diff
@@ -0,0 +1,266 @@
+source('3_run/src/process_glm_output.R')
+source('4_inspect/src/inspect_glm_flows.R')
+source('4_inspect/src/inspect_glm_temps.R')
+
+# These mapped targets are insensitive to whether p3_glm_report.csv is up to
+# date, so make sure you call tar_make(p3_glm_report.csv) first if you want the
+# plots to reflect new model changes
```

Comment:
Yes, they are intentionally decoupled.
And yes, if we did want to link to them, we could indeed refer to the target `p3_glm_report.csv` instead of passing in the file name.